core/storage/classifier/gae_models_test.py

```diff
@@ -64,37 +64,90 @@ def test_create_and_get_new_training_job_runs_successfully(self):
         self.assertEqual(training_job.algorithm_version, 1)
 
     def test_query_new_and_pending_training_jobs(self):
+        offset = 0
```

Comment:
Please keep the original test and also add this test that contains more than 10 models to fetch

Reply:
Done